### PR TITLE
create-modal-order

### PR DESCRIPTION
**Pull request by axel-ledesma-david**
# Summary

- Create component "Modal" with the void content and the children prop
- Add content to the component Modal in Cart component.
- Add styles.
...

# Detail

Create a reusable modal component and aplly it to the orders modal 

# Evidence

image

# Reference.

Link to related documentation.

### CODE REVIEW

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <div className="fixed inset-0 bg-black bg-opacity-30 backdrop-blur-sm flex justify-center items-center">
+      <div className="bg-bg-profile py-5 px-2 roundend flex flex-col justify-center items-center gap-5 w-[40vw] h-[80vh] font-sans shadow-sm">
+        {children}
+      </div>
+    </div>
+  );
+};
+
+Modal.propTypes = {
```
Comment (Fludeo) [inline src/components/Modal/Modal.tsx]: no need to use this library, we already use typescript for types

File: src/components/Cart/Cart.tsx [hunk 2]
```diff
@@ -119,11 +125,80 @@ const Cart = () => {
...
+            className="text-text-paragraph bg-logo-orange relative bottom-24 w-24 rounded-3xl"
+            onClick={() => setModal(!modal)}
+          >
             Pagar
           </button>
         </div>
       </div>
+      {modal ? (
+        <Modal>
+          <div className="justify-center w-[100%] flex flex-col gap-4">
```
Comment (Fludeo) [inline src/components/Cart/Cart.tsx]: we should put the entire form in another component.

**Review by Fludeo — CHANGES_REQUESTED** [review]
pls remove prop-types and use typescript for typing

File: src/components/Modal/Modal.tsx (hunk 1 above)
Comment (axel-ledesma-david) [reply src/components/Modal/Modal.tsx]: done

File: src/components/Cart/Cart.tsx (hunk 2 above)
Comment (axel-ledesma-david) [reply src/components/Cart/Cart.tsx]: done

**Review by Fludeo — APPROVED** [review]

**Pull request merged**